### PR TITLE
Update GitHub Secrets Information in README

### PR DESCRIPTION
**Pull request by mcdonnnj**
# 

## 🗣 Description
This PR updates the section about GitHub Secrets in the README to mention the `terraform-to-secrets` tool in [cisagov/development-guide](https://github.com/cisagov/development-guide). It also appropriately mentions needing appropriate permissions to view/add Secrets to the repository.

## 💭 Motivation and Context
The lack of clear language was brought up in #29 and @felddy created a tool to automagically add desired Secrets from Terraform states to a repository.

## 🧪 Testing
Just automated stuff.

## 🚥 Types of Changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (causes existing functionality to change)

## ✅ Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,16 +24,17 @@ terraform apply
...
-`AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY` environment variables in the
-[repository's secrets](https://github.com/cisagov/skeleton-packer/settings/secrets).
+[repository's secrets](https://help.github.com/en/actions/configuring-and-managing-workflows/creating-and-storing-encrypted-secrets)
+with the new encrypted environment variables. This should be done using the
+`terraform-to-secrets` tool available in the
+[development guide](https://github.com/cisagov/development-guide). Instructions
+for how to use this tool can be found in the
+[Project Setup README](https://github.com/cisagov/development-guide/tree/develop/project_setup).
+
+If you have appropriate permissions for the repository you view existing
```
Comment (jsf9k) [inline README.md]: "...you view..." --> "...you can view..."?

**Review by jsf9k — APPROVED** [review]
Approved, but please see my one comment.

File: README.md [hunk 2]
```diff
@@ -24,16 +24,17 @@ terraform apply
...
-```console
-terraform state show module.iam_user.aws_iam_access_key.key
-```
-
-Take the `id` and `secret` fields from the above command's output and create the
-`AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY` environment variables in the
-[repository's secrets](https://github.com/cisagov/skeleton-packer/settings/secrets).
+[repository's secrets](https://help.github.com/en/actions/configuring-and-managing-workflows/creating-and-storing-encrypted-secrets)
+with the new encrypted environment variables. This should be done using the
+`terraform-to-secrets` tool available in the
```
Comment (dav3r) [inline README.md]: You can make `terraform-to-secrets` link to https://github.com/cisagov/development-guide/tree/develop/project_setup#terraform-iam-credentials-to-github-secrets-

File: README.md [hunk 3]
```diff
@@ -24,16 +24,17 @@ terraform apply
...
+[development guide](https://github.com/cisagov/development-guide). Instructions
+for how to use this tool can be found in the
+[Project Setup README](https://github.com/cisagov/development-guide/tree/develop/project_setup).
+
+If you have appropriate permissions for the repository you view existing
+secrets on the
+[appropriate page][repository settings](https://github.com/cisagov/skeleton-packer/settings/secrets)
```
Comment (dav3r) [inline README.md]: This is probably not how you wanted this link to look here: `[appropriate page][repository settings]`.

File: README.md [hunk 4]
```diff
@@ -24,16 +24,17 @@ terraform apply
...
-
-Take the `id` and `secret` fields from the above command's output and create the
-`AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY` environment variables in the
-[repository's secrets](https://github.com/cisagov/skeleton-packer/settings/secrets).
+[repository's secrets](https://help.github.com/en/actions/configuring-and-managing-workflows/creating-and-storing-encrypted-secrets)
+with the new encrypted environment variables. This should be done using the
+`terraform-to-secrets` tool available in the
+[development guide](https://github.com/cisagov/development-guide). Instructions
+for how to use this tool can be found in the
+[Project Setup README](https://github.com/cisagov/development-guide/tree/develop/project_setup).
```
Comment (dav3r) [inline README.md]: Maybe this `Project Setup README ` link should also go to https://github.com/cisagov/development-guide/tree/develop/project_setup#terraform-iam-credentials-to-github-secrets-

**Review by dav3r — APPROVED** [review]
Approved, but see some minor changes requested. 🔒

**Review by felddy — APPROVED** [review]
Looks good.
Keep on keepin' on!  💪

File: README.md (hunk 1 above)
Comment (mcdonnnj) [reply README.md]: woops should be fixed in 599f2fa

File: README.md (hunk 3 above)
Comment (mcdonnnj) [reply README.md]: Ah missed that when I was rephrasing, should be fixed in 88703b7

File: README.md (hunk 2 above)
Comment (mcdonnnj) [reply README.md]: After our supplementary discussion this makes sense. Added in 
9c31164

File: README.md (hunk 4 above)
Comment (mcdonnnj) [reply README.md]: I've adjusted the phrasing in cff5bc3 to incorporate your suggestion and make it clear that it's going to that section.

**Comment by dav3r** [comment]
Changes look good @mcdonnnj - still approved by me. 👍

**Pull request merged**